_posts/en/newsletters/2020-04-29-newsletter.md

```diff
@@ -0,0 +1,265 @@
+    settlement transaction or getting his refund settlement transaction
+    confirmed, Alice is able to reclaim the 1 BTC she offered Bob in the
+    Alice-Bob HTLC once its 80 block timeout expires.  When Mallory's
+    preimage settlement transaction does eventually confirm, Mallory
+    gets the 1 BTC that Bob offered her in the Bob-Mallory HTLC.  This
+    leaves Bob 1 BTC poorer than he started.
+
+    Several solutions were considered in the thread, but all had
+    problems or involved significant tradeoffs:
+
+    - **Require a mempool:** Bob could use a Bitcoin full node to
+      monitor the Bitcoin P2P relay network and learn about Mallory's
+      settlement transaction.  This seems like a [reasonable amount of
+      extra burden][osuntokun reasonable] since the problem only affects
+      routing nodes (like Bob) and not nodes that only send or receive
+      payments on behalf of themselves, which are more likely to be
+      everyday users running lightweight LN clients on mobile devices.
+      Unfortunately, not all full nodes receive the same transactions
+      even when everything is working perfectly and there are techniques
+      attackers like Mallory can use to [send different conflicting
+      transactions][corallo mempool not guaranteed] to different peers
+      (for example, sending the pinned preimage settlement transaction
+      to known miners but sending a different non-settlement transaction
+      with at least one of the same inputs to non-miner relay nodes).
+
+    - **Beg or pay for preimages:** The relay network could provide
+      [information about conflicts][harding reject] to transaction
+      submitters such as Bob so they wouldn't need to continuously
+      monitor relay themselves.  This still suffers from the problem of
+      bad actors such as Mallory using [targeted relay][corallo targeted
+      relay] to send different transactions to miners and non-miners.
+      Additionally Bob might be able to [pay][harding pay] miners or
+      other third party nodes for the preimage he needs, although this
+      requires some people run additional software and [might not be as
+      easy][zmn ptlcs] to do after the deployment of some proposed
+      upgrades to the LN protocol.
+
+    - **Settlement transaction anchor outputs:** Onchain settlement
+      transactions could be redesigned to spend their value to [anchor
+      outputs][topic anchor outputs] that could be [CPFP fee
+      bumped][topic cpfp] using [CPFP carve-out][topic cpfp carve out].
+      This would require those transactions to be larger (increasing
+      onchain fees) and presigned (reducing flexibility).  This would
+      only affect channels which are unilaterally closed while payments
+      are pending, which is already a situation that that can
+      significantly increase onchain costs and so is something users try
+      to avoid.  As of this writing, this appears to be the most
```

Comment:
By forcing every HTLC to have 2-transactions for every settlement, it would increase the cost of their spending and therefore making them uneconomical, which means they would be considered as dust. And dust HTLCs relies on more trust in your counterparty...